spy_terminal.py

```diff
@@ -27,76 +30,94 @@ def main():
     while x != 'boom':
         if welcome():
             password = set_up()
-            scrambled_message = new_message(codex,count)
+            scrambled_message = new_message(codex, count)
             count = 1
         else:
-            retrieve_message(password, scrambled_message,codex,count)
+            retrieve_message(password, scrambled_message, codex, count)
             scrambled_message = None
             count = 0
         x = input("To self-destruct type 'boom'. Otherwise, hit enter. ")
-    self_destruct(5,'KABOOM', "Yes")
+    self_destruct(5, 'KABOOM', "Yes")
+
 
 '''---------------------SUPPORTING FUNCTIONS ------------------------'''
 
 '''Welcomes the user and directs them to either write a new message or retrieve an existing message.'''
+
+
 def welcome():
     today = date.today()
     print('Welcome Agent. The current date is:', today)
-    desire = input("Enter 'L' to leave a new message. Enter 'R' to retrieve message: ")
+    desire = input(
+        "Enter 'L' to leave a new message. Enter 'R' to retrieve message: ")
     while desire != 'L' and desire != 'R':
-        desire = input("Enter 'L' or leave a new message. Enter 'R' to retrieve message: ")
+        desire = input(
+            "Enter 'L' or leave a new message. Enter 'R' to retrieve message: ")
     if desire == 'L':
         return True
     else:
         return False
 
+
 '''Uses count to check if a message already exists. If so, warns this will be overridden. 
     Takes message and puts it through codex to scramble and then override it.'''
-def new_message(codex,count):
+
+
+def new_message(codex, count):
     if count > 0:
         print("Warning! You already have a message logged. "
               "Your new message will replace it. ")
         time.sleep(1)
         decision = input("Continue? Y/N: ")
         if decision == 'Y':
-            entry = input("Please give me your message. Note: Do not use numbers: ")
+            entry = input(
+                "Please give me your message. Note: Do not use numbers: ")
             scrambled_message = scramble(entry, codex)
             return scrambled_message
```

Comment:
```suggestion
            return scramble(entry, codex)
```
Any time you assign a variable and return it immediately, you should remind yourself to skip the assignment!